pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -77,7 +77,12 @@ var _ = Describe("ImageUpload", func() {
 
 		imagePath = imageFile.Name()
 
-		archiveFilePath = tests.ArchiveFiles("archive", os.TempDir(), imagePath)
+		archiveFile, err := ioutil.TempFile("", "archive")
+		Expect(err).ToNot(HaveOccurred())
+		defer archiveFile.Close()
```

Comment:
Does this file get deleted?

Reply:
in `AfterEach(func() {` line 91: `os.Remove(archiveFilePath)`